src/feedback.py

```diff
@@ -13,16 +13,12 @@ def _ingest_feedback(payload):
     security_event = SecurityEvent(url=payload['url'])
     feedback_ = Feedback(author=payload['author'],
                          feedback_type=FeedBackType[payload['feedback_type']],
+                         url=payload['url'],
                          comments=payload['comments'])
-    g.has_node(security_event).add_unique_node(feedback_)
+    g.has_node(security_event).and_(Traversel.anonymous().add_unique_node(feedback_))
```

Comment:
you can think of  handling  exception in all graph db call Traversel class.

Reply:
I'm not anticipating any handleable exception over here.